src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -596,7 +577,7 @@ std::shared_ptr<GLHEVertSingle> GetSingleBH(EnergyPlusData &state, std::string c
 std::shared_ptr<GLHEVertArray> GetVertArray(EnergyPlusData &state, std::string const &objectName)
 {
     // Check if this instance of this model has already been retrieved
-    for (auto &thisProp : state.dataGroundHeatExchanger->vertArraysVector) {
+    for (auto const &thisProp : state.dataGroundHeatExchanger->vertArraysVector) {
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:599]:(style),[constVariable],Variable 'thisProp' can be declared as reference to const